README.md

```diff
@@ -38,6 +38,7 @@ For more information see the [self-hosted runner security docs](https://docs.git
 | aws_tags              | The AWS tags to use for your runner, formatted as a JSON list. See `README` for more details.                      | false              |         |
 | extra_gh_labels       | Any extra GitHub labels to tag your runners with. Passed as a comma-separated list with no spaces.                 | false              |         |
 | instance_count        | The number of instances to create, defaults to 1                                                                   | false              | 1       |
+| gh_timeout            | The amount of timeout for waiting for a runner to be online by GitHub.                                             | false              | 1200    |
```

Comment:
maybe some clarification here that this is "seen online by GitHub after cloud providers says it has launched"? Not sure the best way to phrase that, but I'm not sure that the current phrasing would be perfectly clear.

Reply:
Fixed in: [abf7339](https://github.com/omsf-eco-infra/gha-runner/pull/22/commits/abf7339bb5ab5273463e926ae4f0df2da053c064)